VmAgent.Core/Dependencies/Interfaces/IFileWriteWrapper.cs

```diff
@@ -8,7 +8,7 @@ namespace Microsoft.Azure.Gaming.VmAgent.Core.Dependencies.Interfaces
     public interface IFileWriteWrapper 
     {
         public void CreateFile(string logFilePath);
-        public void Write(string message);
+        public void Write(string message, string streamType = "stdout");
```

Comment:
In general, default arguments in Interfaces aren't ideal. If you are going to have them, then you should at least get rid of the default arguments on the implementation.
The problem is that it's easy to create a bug when the interface and the implementation have different default values--it's not always intuitive which one the compiler is going to use.